iowrappers/redis_client.go

```diff
@@ -611,32 +611,42 @@ func (r *RedisClient) SavePlanningSolutions(context context.Context, request *Pl
 		return keyGenerationErr
 	}
 
+	// cleans up previous results
+	exists, _ := r.client.Exists(ctx, redisListKey).Result()
+	if exists == 1 {
+		if err := r.client.Del(ctx, redisListKey).Err(); err != nil {
+			Logger.Error(err)
+		}
+	}
+
 	var recordKeys []string
 	for _, record := range request.PlanningSolutionRecords {
 		solutionRedisKey := strings.Join([]string{TravelPlanRedisCacheKeyPrefix, record.ID}, ":")
 		json_, err := json.Marshal(record)
 		if err != nil {
-			return err
+			Logger.Error(err)
+			continue
```

Comment:
why `continue` everywhere ? Is it to make sure we go through all the `redisKeys` ?

Reply:
Correct. We don't want return an error and miss the rest of the plans like we did before.